jvb/src/main/resources/reference.conf

```diff
@@ -294,6 +294,11 @@ videobridge {
     }
   }
 
+  # The experimental multiple streams per endpoint support
+  source-name-signaling {
```

Comment:
Should we add a more generic "multi-stream" config context?

Reply:
can we keep it as is for now with a note here that it's experimental and subject to be removed without warning?
I can't come up with a good name, because can't tell what properties will be there yet. I imagine maybe something around default behavior when picking streams, but the "multi-stream" may not be the best name at that point.

Reply:
or I will change it, even if this may change it's better to start with something more generic